pyproject.toml

```diff
@@ -40,7 +40,7 @@ docs = ["Sphinx==7.1.2","sphinx-rtd-theme==2.0.0","myst-parser==3.0.1"]
 quality = ["black", "ruff", "hf_doc_builder@git+https://github.com/huggingface/doc-builder.git"]
 
 [build-system]
-requires = ["setuptools>=62.0.0"]
+requires = ["setuptools>=62.0.0", "wheel", "build"]
```

Comment:
These packages need not be specified here. They will not be installed automatically before running build. They need to be installed manually.

Reply:
Agree, removing them from here.